lib/LaTeXML/Package.pm

```diff
@@ -430,9 +430,14 @@ my @rmletters = ('i', 'v', 'x', 'l', 'c', 'd', 'm');    # [CONSTANT]
 
 sub roman_aux {
```

Comment:
Side comment, the `roman` functions really feel like `Util`-level pieces of code, rather than `Package.pm` bits.